security.php

```diff
@@ -97,9 +97,13 @@ function wpcom_vip_login_limit_dont_show_login_form() {
 }
 add_action( 'login_form_login', 'wpcom_vip_login_limit_dont_show_login_form' );
 
+
 function wpcom_vip_login_limit_xmlrpc_error( $error, $user ) {
-	if ( is_wp_error( $user ) && ERROR_CODE_LOGIN_LIMIT_EXCEEDED == $user->get_error_code() )
+	static $login_limit_error;
```

Comment:
Can we add some clarifying comments on why this is necessary, for the World of Tomorrow?

Reply:
Sure. Was thinking a test would also be helpful for context.

Reply:
Hopefully this comment makes sense.